include/tvm/relay/interpreter.h

```diff
@@ -140,6 +140,28 @@ struct TensorValueNode : ValueNode {
 
 RELAY_DEFINE_NODE_REF(TensorValue, TensorValueNode, Value);
 
+/*! \brief An ADT constructor value. */
+class ConValue;
+
+struct ConValueNode : ValueNode {
```

Comment:
will name `ConstructorValueNode` or [CtorValueNode](
https://en.m.wikipedia.org/wiki/Constructor_(object-oriented_programming)) be better?

Reply:
I'll go with ConstructorValueNode for consistency